health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -1173,20 +1163,116 @@
+                  }}
+                  showBottom
+                  style={{
+                    whiteSpace: "nowrap",
+                    width: "auto",
+                  }}
+                  title=""
+                />
+
+                <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center", gap: "1rem" }}>
+                  <div>
+                    <label style={{ marginRight: "5px" }}>Rows per page:</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)